src/Microsoft.AspNet.Mvc.Core/UrlHelperExtensions.cs

```diff
@@ -57,6 +57,37 @@ public static string Action([NotNull] this IUrlHelper helper, string action, str
             return helper.Action(action, controller, values, protocol, host, fragment: null);
         }
 
+        /// <summary>
+        /// Generates a fully qualified or absolute URL for an action method by using the specified action name,
+        /// controller name, route values, protocol to use, host name and fragment.
+        /// </summary>
+        /// <param name="action">The name of the action method.</param>
+        /// <param name="controller">The name of the controller.</param>
+        /// <param name="values">An object that contains the parameters for a route.</param>
+        /// <param name="protocol">The protocol for the URL, such as "http" or "https".</param>
+        /// <param name="host">The host name for the URL.</param>
+        /// <param name="fragment">The fragment for the URL.</param>
+        /// <returns>The fully qualified or absolute URL to an action method.</returns>
```

Comment:
The other Action overloaded extension methods don't have this xml docs. I'll copy these comments to those methods before checking in them.